aim/db/hashtree_db_listener.py

```diff
@@ -28,6 +28,7 @@
 from aim import config as aim_cfg
 from aim import tree_manager
 
+ACTION_LOG_THRESHOLD = 1000
```

Comment:
LGTM but just wondering should we make this threshold configurable just in case user complains either not seeing the logs (but the system is already slow) or seeing too many logs while system is still operational....

Reply:
If it were affecting a behavior I'd say yes - but in this case it's just a log message. The threshold number I picked is a bit arbitrary, but in a normal working setup it shouldn't get triggered.